Modules/Messages/src/Messages.Infrastructure/Persistence/Database/Repository/RelationshipsRepository.cs

```diff
@@ -22,4 +23,12 @@ public RelationshipsRepository(MessagesDbContext dbContext)
             .Select(r => r.Id)
             .FirstOrDefaultAsync();
     }
+
+    public Task<Relationship?> FindRelationship(RelationshipId id, CancellationToken cancellationToken, bool track = false)
+    {
+        return _dbContext.Relationships
+            .AsNoTracking()
```

Comment:
Mhh, you always use `AsNoTracking`, no matter what you pass as `track`. Though, when I think about it: we will never ever update a Relationship from the Messages module anyway. So you might as well just remove this parameter.

Reply:
Probably because I copied the signature from another place, I agree that no tracking is required for relationships from the Messages module. 

Changes made.